doc/install.rst

```diff
@@ -259,12 +259,17 @@ library (``libgmt``).
 This can happen if you have multiple versions of GMT installed.
 
 You can tell PyGMT exactly where to look for ``libgmt`` by setting the
-``GMT_LIBRARY_PATH`` environment variable.
-This should be set to the directory where ``libgmt.so``, ``libgmt.dylib`` or
-``gmt.dll`` can be found for Linux, macOS and Windows, respectively.
-e.g., on a command line, run::
+``GMT_LIBRARY_PATH`` environment variable to the directory where ``libgmt.so``,
+``libgmt.dylib`` or ``gmt.dll`` can be found for Linux, macOS and Windows,
+respectively.
+
+For Linux/macOS, adding the following line to your shell configuration file
+(usually `~/.bashrc` for Bash on Linux and `~/.zshrc` for Zsh on macOS)::
```

Comment:
Should we use the tab thing for Linux/macOS/Windows instructions? Similar to what is currently used for conda/mamba?

Reply:
I don't think it's necessary.